### ISSUE TITLE
Task Scheme incorrect error on multi-line jinja2

### ISSUE THREAD

**Issue by nlvw** [ansible/ansible-lint#2772]
The yaml below works correctly when run but this schema, used with ansible-lint, throws an error.

<img width="453" alt="image" src="https://user-images.githubusercontent.com/5723368/200680353-f2d2bf2d-af32-4633-b53e-83993d652a1c.png">

```yaml
- name: update basic package list
  set_fact:
    pkgs_basic: "{{ pkgs_basic + lookup('ansible.builtin.vars', item) }}"
  with_items: >-
    {{ query(
      'ansible.builtin.varnames',
      '^pkgs_basic_.*$',
      '^pkgs_el' + ansible_distribution_major_version + '$',
      '^pkgs_el' + ansible_distribution_major_version + '_.*$'
    ) }}
```

ansible = 6.5.0
ansible-lint = 6.8.6

```console
$ ansible --version
ansible [core 2.13.5]
  config file = /home/nvonwolf/nmsu/ansible/ansible-ict-systems-monolithic/ansible.cfg
  configured module search path = ['/home/nvonwolf/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.10/site-packages/ansible
  ansible collection location = /home/nvonwolf/nmsu/ansible/ansible-ict-systems-monolithic/.cache/collections:/home/nvonwolf/nmsu/ansible/ansible-ict-systems-monolithic/collections
  executable location = /usr/local/bin/ansible
  python version = 3.10.7 (main, Sep  7 2022, 00:00:00) [GCC 12.2.1 20220819 (Red Hat 12.2.1-1)]
  jinja version = 3.1.2
  libyaml = True
```

**Comment by ssbarnea** [comment]
Apparently the bug originates in the validation library we use, https://github.com/python-jsonschema/jsonschema/issues/1023

**Comment by nlvw** [comment]
Thanks for the quick fix on this!
